e2e/test/scenarios/models/reproductions.cy.spec.js

```diff
@@ -1861,3 +1861,52 @@ describe("issue 42355", () => {
       .should("be.visible");
   });
 });
+
+describe("cumulative count - issue 33330", () => {
```

Comment:
nit: I thought we name describe exactly `issue 33330`

Reply:
I don't think it really matters.
There's 2 sub-repros here so it made sense to hint what it is all about.